bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -234,12 +234,11 @@ void PreProcessor::onMessage<PreProcessReplyMsg>(PreProcessReplyMsg *msg) {
     clientEntry->clientReqInfoPtr->handlePreProcessReplyMsg(preProcessReplyMsg);
     result = clientEntry->clientReqInfoPtr->getPreProcessingConsensusResult();
   }
-  MDC_CID_PUT(GL, clientEntry->clientReqInfoPtr->getPreProcessRequest()->getCid());
   switch (result) {
     case CONTINUE:  // Not enough equal hashes collected
       return;
     case COMPLETE:  // Pre-processing consensus reached
-      finalizePreProcessing(clientId, reqSeqNum);
+      finalizePreProcessing(clientId, reqSeqNum, "" /* cid will be added later */);
```

Comment:
Please move the comment out from the parentheses.

Reply:
done